src/server/pps/server/api_server.go

```diff
@@ -1104,6 +1106,14 @@ func (a *apiServer) watchJobCompletion(ctx context.Context, job *pps.Job, jobCom
 	})
 }
 
+func (a *apiServer) numWorkers(ctx context.Context, deploymentName string) (int, error) {
```

Comment:
I believe we have a `GetExpectedNumWorkers` function that returns the number of works using the parallelism spec.